pkg/api/version.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+// Version is set in the build process.
+var Version string
```

Comment:
it is actually not needed to make the member public. the build tags can also set a package scoped variable.

Reply:
It is actually used it in the `debugapi` package so it must be exported.